src/shared/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
```

Comment:
Do we really need a separate button component?  Can this simple be a tag on the home page?